tools/secure_notebook.py

```diff
@@ -66,16 +78,33 @@ def create_self_signed_cert(cert_dir, keyfile, certfiile):
     except ConfigFileNotFound:
         config = Config()
 
-    config.NotebookApp.password = pw
+    yield config
 
     with io.open(os.path.join(jupyter_config_dir(), 'jupyter_notebook_config.json'), 'w') as f:
```

Comment:
That usually is handled at jupyter_ore level that config dir should be 0o700, the password is sha-hashed, so not **that** sensible.